layers/+lang/lua/funcs.el

```diff
@@ -24,7 +24,11 @@
     "sr" 'lua-send-region
     "'" 'lua-show-process-buffer)
   (pcase lua-backend
-    (`lsp-emmy (spacemacs//lua-setup-lsp-emmy))))
+    ('lsp-emmy-lua (spacemacs//lua-setup-lsp-emmy-lua))
+    ('lsp-lua-language-server (spacemacs//lua-setup-lsp-lua-language-server))
+    ('lsp-lua-lsp (spacemacs//lua-setup-lsp-lua-lsp))
+    (_ (when (configuration-layer/layer-used-p 'lsp)
```

Comment:
The catuch-all case `_` should better throw an error instead.
For example,
```elisp
(_ (user-error "Unexpected value of `lua-backend', %s" lua-backend))
```

Reply:
Issue is resolved in new submition.